package.json

```diff
@@ -17,6 +17,7 @@
   "workspaces": [
     "packages/annotated/*",
     "packages/mephisto-task",
+    "packages/mephisto-worker-experience",
```

Comment:
we'll have to rename to `mephisto-worker-addons`. happy to accept that either in this PR or a quick follow up

Reply:
Yeah, I did it in this pr